components/notifier/spec/models/notifier/template_spec.rb

```diff
@@ -2,6 +2,18 @@
 
 module Notifier
   RSpec.describe Template, type: :model do
-    pending "add some examples to (or delete) #{__FILE__}"
+    # pending "add some examples to (or delete) #{__FILE__}"
+
+    describe '#attributes' do
+      subject { FactoryGirl.create(:notifier_template) }
```

Comment:
Factory is not been picking up from engine. if we define in application it is been picking up. need to figure out.